guides/common/modules/proc_configuring-podman-to-trust-the-ca.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Configuring_podman_to_trust_the_CA_{context}"]
+= Configuring Podman and Docker to trust the Certificate Authority
+
+Podman uses two paths to locate the CA file, namely, `/etc/containers/certs.d/` and `/etc/docker/certs.d/`.
+
+* Copy the root CA file to one of these locations, with the exact path determined by the server hostname, and naming the file `ca.crt`:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ sudo cp rootCA.pem /etc/containers/certs.d/quay-server.example.com/ca.crt
```

Comment:
I'm not sure how much we use `sudo` in advice. The convention of `$` means it's non-root and `sudo` implies root. So The following would be equal:
```suggestion
# cp rootCA.pem /etc/containers/certs.d/quay-server.example.com/ca.crt
```

Then I also think you should drop `quay` from the example hostname. We have no relation to Quay and it works with everything. It could also be a Pulp server.

Lastly, the directory `/etc/containers/certs.d/quay-server.example.com` may not exist so a `mkdir` or `mkdir -p` may be good advice.

(note this is repeated in all examples, I'm not commenting everywhere inline)

Reply:
+1  

I'd also change it to  satellite-server.example.com

Reply:
What about {project-context}-server.example.com ?

Reply:
or  satellite.example.com  as long as it indicates its the satellite server (or capsule server)

Reply:
I'm leaning to `server.example.com` or `hostname.example.com` to clearly indicate to the user that's how the mechanism works.